pkgs/applications/misc/gnss-sdr/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchFromGitHub
+, armadillo
+, blas
```

Comment:
A fair concern, but without them the build fails with one of these two messages:

>      The LAPACK library has not been found.
>      You can try to install it by typing:
>      sudo apt-get install liblapack-dev
>     CMake Error at CMakeLists.txt:762 (message):
>       LAPACK is required to build gnss-sdr

or

>      The BLAS library has not been found.
>      You can try to install it by typing:
>      sudo apt-get install libopenblas-dev
>     CMake Error at CMakeLists.txt:783 (message):
>       BLAS is required to build gnss-sdr

Also, I tested building with both `openblasCompat` and `openblas`, and the configure script failed to detect either one. It only accepts `blas`. Which is funny since the Debian package it helpfully suggests installing is for openblas...

I don't understand any of the relevant code well enough to guess what the right fix is. In particular, I haven't worked with lapack/blas before, but I'm also not super comfortable with cmake. Any suggestions?